### ISSUE TITLE
Focus halo must have color contrast ratio of 3:1 to meet WCAG 2.1 success criteria 1.4.11 non-text contrast

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-design-system-documentation#359]
# Bug Report

- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened
The yellow focus halo we are using throughout VA.gov for visually representing the element with keyboard focus is not future proof for WCAG 2.1. Specifically the focus halo does not meet [SC 1.4.11 Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html). This success criteria states:
> any visual information necessary to indicate state, such as whether a component is selected or focused must also ensure that the information used to identify the control in that state has a minimum 3:1 contrast ratio.

/cc @KevinMHoffmanUSDS & @artsymartha68 for visibility

## What I expected to happen
- [ ] The focus halo has at least a 3:1 contrast ratio with the background color
- [ ] Focus halo meets the 3:1 contrast ratio on light, dark, and shaded backgrounds
- [ ] Focus halo should meet 3:1 contrast ratio in high-contrast scenarios ( https://www.a11yproject.com/posts/2020-01-23-operating-system-and-browser-accessibility-display-modes/ )

## Reproducing
![Screen Shot 2020-12-14 at 5 04 42 PM](https://user-images.githubusercontent.com/934879/102146796-3f38bc80-3e2f-11eb-8fd3-55f7cb3ad5d6.png)

## Acceptance Criteria for Fix
- [ ] Create and/or modify unit tests for this component to include an aXe scan
- [ ] The focus halo has at least a 3:1 contrast ratio with the background color
- [ ] Focus halo meets the 3:1 contrast ratio on light, dark, and shaded backgrounds
- [ ] Focus halo should meet 3:1 contrast ratio in high-contrast scenarios ( https://www.a11yproject.com/posts/2020-01-23-operating-system-and-browser-accessibility-display-modes/ )

**Comment by caw310** [comment]
This was discussed during the DC meeting 12/18. 
Action items: 
- Fix CSS to meet contrast ratio requirements. Set 2 different colors. 
- Two colors - one on light background and one on dark background

Will put this in the DST backlog to be prioritized.

**Comment by noahgelman** [comment]
Spoke with Ryan Hurwell, he will give me the stats on the new focus ring next week for me to implement.

**Comment by artsymartha68** [comment]
I'm glad we can have 2 different colors! This would be all but impossible otherwise.

**Comment by artsymartha68** [comment]
![FocusColorsFinal](https://user-images.githubusercontent.com/47792423/131385750-b2839337-b8d6-4f10-b482-14b85de314ee.png)

Ryan and I decided to go with #988530 for the light background areas. The ultimate solution would be an inverted look a la GOV.UK but this is just to mitigate the current problematic yellow-on-light situation.

**Comment by noahgelman** [comment]
@rtwell Hey Ryan, just wanted to make sure you were in the loop for this

**Comment by k80bowman** [comment]
@noahgelman just checking in. Is this something you might be able to get back to soon?

**Comment by noahgelman** [comment]
Yes, I'm looking to get this wrapped up soon

**Comment by caw310** [comment]
This item was discussed in the 2/25 DSC meeting. Here are the action items: 

- Move forward with a demo of it or pull request that makes this change. 
- Apply the white inner and olive outer highlights

**Comment by bkjohnson** [comment]
I wrote a [new issue](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/654) to capture the change in work. We may want to close this one now.
